src/targets/node/nodeSourcePathResolver.ts

```diff
@@ -20,11 +20,18 @@ export class NodeSourcePathResolver extends SourcePathResolverBase<IOptions> {
       return '';
     }
 
-    const withBase = path.resolve(this.options.basePath, this.applyPathOverrides(url));
-    return this.rebaseRemoteToLocal(withBase);
+    const modified = this.applyPathOverrides(url);
+
+    if (modified !== url) {
```

Comment:
Why this change?

Reply:
Without this change, this function would return the incorrect absolute path for the node internals. It would apply the path overrides, which would come back unchanged, and then tack on the base path, resulting in something that looked like "C:\projects\appbase\internal\modules\cjs\loader.js", which would throw the logic off since I'm basing filtering of node's internals on whether the source has an absolute path.

Reply:
I think the sourceMapPathOverrides shouldn't apply at all to paths that we know are node internals. We don't want to break this if one pattern accidentally applies.